lib/game_looper.rb

```diff
@@ -0,0 +1,56 @@
+require 'pry'
+
+class GameLooper
+  # private
+
+  def initialize(console:, display:, prompt:, board:, players:)
+    @console = console
+    @welcome_message = welcome_message
+    @display = display
+    @prompt = prompt
+    @board = board
+    @players = players
+  end
+
+  def loop
+    take_turns(players) until game_is_over
```

Comment:
Very concise line here 👍 
Reads just like you'd describe a game "the players take turns until the game is over"